tests/TestCIT_KCI.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+import numpy as np
+
+import causallearn.utils.cit as cit
+
+
+class TestCIT_KCI(unittest.TestCase):
+    def test_Gaussian(self):
+        X = np.random.randn(300, 1)
+        X1 = np.random.randn(300, 1)
+        Y = np.concatenate((X, X), axis=1) + 0.5 * np.random.randn(300, 2)
+        Z = Y + 0.5 * np.random.randn(300, 2)
+
+        cit_CIT = cit.CIT(data=np.hstack((X, X1, Y, Z)), method='kci')
+
+        pvalue = cit_CIT(0, 1)
+        self.assertTrue(pvalue > 0.01)  # X and X1 are independent
```

Comment:
this looks very weak. For independent variables, we should expect the p-value is much larger than 0.01? What are the true p-values returned here?

actually I believe under the null, p-value should follow uniform distribution?

hmmm, this reminds me to think about how to write good test for hypothesis testing --- probably we need to some distribution level tests to be robust. Probably we can discuss in the group chat. :)